<!--xml-->
<launch>

  <!-- global settings -->
  <param name="faceRecog" type="bool" value="true" />
  <param name="locationRecog" type="bool" value="true" />
  <param name="objectRecog" type="bool" value="true" />

  <param name="speechLibPath" type="string" value="$(find mmfpspkg)/library/lib_speech/"/>
  <param name="faceLibPath" type="string" value="$(find mmfpspkg)/library/lib_face/" />
  <param name="locationLibPath" type="string" value="$(find mmfpspkg)/library/lib_location/"/>
  <param name="retailLibPath" type="string" value="$(find mmfpspkg)/library/lib_retail/"/>
  <param name="bnetLibPath" type="string" value="$(find mmfpspkg)/library/lib_bnet/"/>
  <param name="responderLibPath" type="string" value="$(find mmfpspkg)/library/lib_responder/"/>

  <!-- face recognition node -->
  <node name="face" pkg="mmfpspkg" type="face" output="screen" >
    <param name="faceCascadePath" type="string" value="$(find mmfpspkg)/library/lib_face/haarcascades/haarcascade_frontalface_alt.xml"/>
	<param name="leftEyeCascadePath" type="string" value="$(find mmfpspkg)/library/lib_face/haarcascades/haarcascade_mcs_lefteye.xml"/>
	<param name="rightEyeCascadePath" type="string" value="$(find mmfpspkg)/library/lib_face/haarcascades/haarcascade_mcs_righteye.xml"/>
	<param name="modelPath" type="string" value="$(find mmfpspkg)/library/lib_face/people/model.xml"/>
	<param name="dataPath" type="string" value="$(find mmfpspkg)/library/lib_face/people"/>
  </node>

</launch>
